app/controllers/reports_controller.rb

```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
 
 class ReportsController < ApplicationController
-  before_action :set_report_of_current_user, only: %i[edit update destroy]
+  before_action :set_current_user_report, only: %i[edit update destroy]
 
   # GET /reports or /reports.json
   def index
-    @reports = Report.order(created_at: 'DESC').page(params[:page])
+    @reports = Report.order({ created_at: 'DESC' }, :id).page(params[:page])
```

Comment:
idを含めるなら

```suggestion
    @reports = Report.order(id: :desc).page(params[:page])
```

でもほぼ同じ結果になりそうです

Reply:
なるほど、今回の例ならほぼ`idの降順 = created_atの降順`なので`id`で並べた方が記述が少ないです🤔
更新日等で並べるなら`id`のように別の条件も使って一意な順番にする必要があるのですね。